recipes-devtools/python/python3-jepture_git.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "A simple python library to quickly capture images on the Jetson."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=4f0a72805ccc91536540a1cf89a935ec"
```

Comment:
Since you're also pulling in the Jetson MMAPI sources, the above settings should also cover those files.

Reply:
Done